x-pack/plugins/ml/public/application/jobs/new_job/pages/new_job/page.tsx

```diff
@@ -63,7 +63,7 @@ export const Page: FC<PageProps> = ({ existingJobsAndGroups, jobType }) => {
   const jobCreator = useMemo(
     () =>
       jobCreatorFactory(jobType)(
-        mlContext.currentDataView,
+        mlContext.selectedDataView,
```

Comment:
also here, would be nice to rename `mlContext` to avoid confusion

Reply:
Renamed them here [`7d120b9` (#159625)](https://github.com/elastic/kibana/pull/159625/commits/7d120b95964bbd3237adb6a106e47f34a81bab6b)